### ISSUE TITLE
[Laje2013] Robust timing and motor patterns by taming chaos in recurrent neural networks

### ISSUE THREAD

**Issue by mxochicale** [mxochicale/arXiv#27]
## 169 Google Scholar Citations by 7april2018
https://scholar.google.co.uk/scholar?gws_rd=cr&dcr=0&um=1&ie=UTF-8&lr&cites=5173393963113158972

FILE: https://scholar.google.co.uk/scholar?cluster=5173393963113158972&hl=en&as_sdt=0,5&sciodt=0,5

ReScience: https://github.com/ReScience-Archives/Vitay-2016/tree/master/article
PDF: https://www.julien-vitay.net/pdf/Vitay2016.pdf

# BibTex-File
```
TY  - JOUR
AU  - Laje, Rodrigo
AU  - Buonomano, Dean V
TI  - Robust timing and motor patterns by taming chaos in recurrent neural networks
JO  - Nature Neuroscience
PY  - 2013/05/26/online
VL  - 16
SP  - 925
EP  -
PB  - Nature Publishing Group, a division of Macmillan Publishers Limited. All Rights Reserved.
SN  -
UR  - http://dx.doi.org/10.1038/nn.3405
L3  - 10.1038/nn.3405
M3  - Article
L3  - https://www.nature.com/articles/nn.3405#supplementary-information
ER  -
```

# Robust timing and motor patterns by taming chaos in recurrent neural networks 

## Author(s) Surname Foreanem / Website
Laje Rodrigo 
https://scholar.google.co.uk/citations?user=2avORIAAAAAJ

Buonomano Dean V
https://scholar.google.co.uk/citations?user=W-2MXuEAAAAJ

## Results

# RAW REVIEWS on date?

INTERESTING SECTION: 

Largest Lyapunov exponent (LLE) estimates.
We estimated the local LLE (λ, also known as finite-time LLE) in a manner similar to that described previously28 with some improvements described elsewhere53,54 (see Supplementary Fig. 8). We computed the distance between perturbed and unperturbed trajectories in state space as a function of time, then found an interval in which the log(distance) versus time plot is linear with a well-defined slope. To this end, the network was first run with random initial conditions and no noise to get a fiducial trajectory x(t) in 800-dimensional state space. Ten segments of 1,000-ms duration were extracted from the fiducial trajectory, which served as the unperturbed trajectories. The first segment x1(t) started 100 ms after the input went off, at state x(100); the second x2(t), third x3(t), etc. segments started 100 ms after the previous, at states x(200), x(300), etc. The perturbed trajectories were obtained as follows. Each segment was perturbed at its initial time point (for instance, for the first segment, the state x(100) was perturbed) by a uniform-noise vector of size 10−7 and then the network was run for 1,000 ms; the perturbation was performed ten times, leading to ten perturbed trajectories yij(t) for each unperturbed segment xi(t). We then computed the average di(t) of the Euclidean distances between each of the ten perturbed trajectories and the unperturbed trajectory (for each of the ten segments) in 800-dimensional state space as a function of time (0–1,000 ms):

￼
We then normalized it to the initial distance (0 ms), that is, the size of the perturbation, computed the logarithm and averaged across all segments: ￼. This procedure was repeated ten times for each of the ten networks. We visually searched for a portion in which all ten repetitions have a linear shape with a well-defined slope of at least 300-ms duration in 100–900-ms range; the average slope of the linear fits was the local LLE estimate for each network at each condition (pre-training, post-training, post-training outside the trained interval). In Figure 6a, for visualization purposes, we plot the average of the ten repetitions for each network: ￼. In the condition post-training outside the trained interval, the fiducial trajectory started 8 s after the input went off (6 s after the end of the training window). Note that any trajectory not converging to a fixed point will have at least one zero Lyapunov exponent (in the direction of the flow); if the trajectory is stable, then it will be the largest Lyapunov exponent, all other exponents being negative.
